tests/ui/suggestions/into-convert.rs

```diff
@@ -13,7 +13,6 @@ fn main() {
 
     let z: AtomicU32 = 1;
     //~^ ERROR mismatched types
-    //~| HELP call `Into::into` on this expression to convert `{integer}` into `AtomicU32`
```

Comment:
annoying, we don't have a placeholder for `{integer}` :thinking: maybe we should instead evaluate and then check whether anything has been constrained?